ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -126,7 +126,9 @@ open class DrawerController: UIViewController {
     @objc open var backgroundColor: UIColor = Colors.Drawer.background {
         didSet {
             useCustomBackgroundColor = true
-            view.backgroundColor = backgroundColor
+            if isViewLoaded {
+                view.backgroundColor = backgroundColor
+            }
```

Comment:
What happens in the case of the view NOT being loaded? Will there be unexpected behavior? Should you have some fallback behavior for setting the background color when the view does end up getting loaded, like a lazy set? No idea if that's possible though.

Reply:
If the view isn't loaded, setting the background color will force the view to load which has led to unwanted behavior from a partner as described in the issue linked in the description. We don't need any fallback behavior since the background color is set in `viewDidLoad` anyway.